pwndbg/lib/config.py

```diff
@@ -97,11 +122,20 @@ def __init__(self):
         self.params = {}
         self.triggers = collections.defaultdict(lambda: [])
 
-    def add_param(self, name, default, docstring, scope="config"):
+    def add_param(
```

Comment:
The number of parameters here is growing large. I'm wondering if we should force callers of this function to use keyword arguments for the optional fields we've added. We can do this like this:
```python
def add_param(
        self,
        name,
        default,
        docstring,
        *,
        help_docstring=None,
        param_class=None,
        enum_sequence=None,
        scope="config",
    ):
```

Everything after the * is required to be a keyword argument.

Reply:
I think this is a good idea!
I'll update it to what you suggest.